be/src/runtime/load_stream_writer.cpp

```diff
@@ -130,6 +130,41 @@ Status LoadStreamWriter::append_data(uint32_t segid, uint64_t offset, butil::IOB
     return file_writer->append(buf.to_string());
 }
 
+Status LoadStreamWriter::append_inverted_index_data(uint32_t segid, uint64_t offset,
```

Comment:
append_inverted_index_data() can be merged to append_data() by adding a type argument and use a std::vector<io::FileWriterPtr>* to reference _segment_file_writers or _inverted_file_v2_writers based on type.

Reply:
done